Description: Disable dlopen feature following wayland-backend
 Honestly I don't know why the latter happened, but it breaks this.
Forwarded: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-04-13
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,5 +35,5 @@
-dlopen = [
-    "native_lib",
-    "wayland-backend/dlopen",
-    "wayland-backend/dlopen",
-]
+#dlopen = [
+#    "native_lib",
+#    "wayland-backend/dlopen",
+#    "wayland-backend/dlopen",
+#]
